R/importPSIMITAB.R

```diff
@@ -0,0 +1,107 @@
+library(stringr)
+
+importNet.STRING <- function(fName,
+                             cutoffType = "xN",
+                             val,
+                             experimentType = getOptions("rete.defaultPPI"),
+                             taxID = "9606",
+                             silent = FALSE,
+                             noLog = FALSE) {
```

Comment:
I was going with what was here http://steipe.biochemistry.utoronto.ca/abc/students/index.php/User:Boris/BCB420_2017_tasks/IMPORT-N.MITAB

I can change to writeLog?